ntbs-service/appsettings.json

```diff
@@ -7,13 +7,15 @@
     "ServiceGroupAdPrefix": "App.Auth.NIS.NTBS.Service"
   },
   "AdfsOptions": {
-    "AdfsUrl": "https://fs-ntbs.uksouth.cloudapp.azure.com"
+    "AdfsUrl": "https://fs-ntbs.uksouth.cloudapp.azure.com",
+    "CookieExpireTimeSpan": "1.00:00:00"
   },
   "AzureAdOptions": {
     "ClientId": "<Provided by deployment secrets>",
     "ClientSecret": "<Provided by deployment secrets>",
     "Authority": "<Provided by deployment secrets>",
     "CallbackPath": "/signin-oidc",
+    "CookieExpireTimeSpan": "1.00:00:00",
```

Comment:
This is really independent of the type of AD we're using, so I think it would be better appearing once in the common `AdOptions` rather than in both ADFS and AzureAD.

Reply:
I also would have been tempted to make this a bit less error prone by making it an integer called `MaxSessionLifetimeInHours` or something like that. It's not a big concern, but it makes it harder for the person changing it to accidentally mess up the format and thereby break the app.

Reply:
Yeah I agree, thanks for that!